Pradeo/pradeo-mtd/ingest/parser.yml

```
@@ -0,0 +1,184 @@
+          pradeo.device.serialNumber: '{{json_event.message.content.deviceCorrelation.device.serialNumber}}'
+          pradeo.device.name: '{{json_event.message.content.deviceCorrelation.device.name}}'
+          pradeo.device.email: '{{json_event.message.content.deviceCorrelation.device.email}}'
+          pradeo.device.imei: '{{json_event.message.content.deviceCorrelation.device.imei}}'
+          pradeo.device.byod: '{{json_event.message.content.deviceCorrelation.device.byod}}'
+          pradeo.device.declaredOperatingSystem: '{{json_event.message.content.deviceCorrelation.device.declaredOperatingSystem}}'
+          pradeo.device.declaredOperatingSystemVersion: '{{json_event.message.content.deviceCorrelation.device.declaredOperatingSystemVersion}}'
+          pradeo.device.declaredOperatingSystemSecurityPatchDate: '{{json_event.message.content.deviceCorrelation.device.declaredOperatingSystemSecurityPatchDate}}'
+          pradeo.device.declaredModel: '{{json_event.message.content.deviceCorrelation.device.declaredModel}}'
+          pradeo.device.lastConnection: '{{json_event.message.content.deviceCorrelation.device.enrollmentStatus.lastConnection}}'
+          pradeo.device.coupled: '{{json_event.message.content.deviceCorrelation.device.enrollmentStatus.coupled}}'
+          pradeo.device.mdmId: '{{json_event.message.content.deviceCorrelation.device.emmDeviceInfo.externalId}}'
+          pradeo.device.emm: '{{json_event.message.content.deviceCorrelation.device.emmDeviceInfo.emm}}'
+          pradeo.policy.id: '{{json_event.message.content.deviceCorrelation.detection.policy.id}}'
+          pradeo.policy.name: '{{json_event.message.content.deviceCorrelation.detection.policy.name}}'
+          pradeo.correlation.applicationThreatLevel: '{{json_event.message.content.device.correlation.applicationThreatLevel}}'
+          pradeo.correlation.networkThreatLevel: '{{json_event.message.content.device.correlation.networkThreatLevel}}'
+          pradeo.correlation.systemThreatLevel: '{{json_event.message.content.device.correlation.systemThreatLevel}}'
+          pradeo.correlation.matchedSystemStatusLevels: '{{json_event.message.content.device.correlation.matchedSystemStatusLevels}}'
+          pradeo.correlation.matchedNetworkStatusLevels: '{{json_event.message.content.device.correlation.matchedNetworkStatusLevels}}'
+  compliance:
+    actions:
+      - filter: '{{json_event.message.type == "DeviceApplicationComplianceUpdated"}}'
+        set:
+          pradeo.device.id: '{{json_event.message.content.deviceApplicationCompliance.deviceApplication.device.id}}'
+          pradeo.device.serialNumber: '{{json_event.message.content.deviceApplicationCompliance.deviceApplication.device.serialNumber}}'
+          pradeo.device.name: '{{json_event.message.content.deviceApplicationCompliance.deviceApplication.device.name}}'
+          pradeo.device.email: '{{json_event.message.content.deviceApplicationCompliance.deviceApplication.device.email}}'
```

[`user.email`](https://www.elastic.co/guide/en/ecs/8.11/ecs-user.html#field-user-email) is more suitable.
```suggestion
          user.email: '{{json_event.message.content.deviceApplicationCompliance.deviceApplication.device.email}}'
```